ultranest/popstepsampler.py

```diff
@@ -536,8 +536,219 @@ def __next__(
         else:
             return None, None, None, nc
 
+class PopulationSimpleSliceSampler():
+    """
+       Vectorized Slice sampler without stepping out procedure.
+       In Comparison the PopulationSliceSampler, the sampler calls the likelihood on
+       batch of points of the same size.
+
+       Sliced are defined by the generate_direction function on a interval defined
+       around the current point. The centred interval has the width of the scale parameter.
+       Slices are then shrink towards the current point until a point is found with a
+       likelihood above the threshold.
+
+       A slice can be searched with more than one point at a time. In that case, we 
+       read points as if they were the next selected each after the other. For a points
```

Comment:
This part should be explained in PopulationSliceSampler and here, consistently. 

"To ensure similar behaviour as if slice sampling were run in sequence, the next independent point is taken from chains in order (i.e., waiting for completion). In case the likelihood in the next step is raised and the current slice state is no longer in the likelihood-restricted prior, the chain is discarded."